src/ffaudio/ffaudio-core.cc

```diff
@@ -431,8 +431,13 @@ bool FFaudio::read_tag (const char * filename, VFSFile & file, Tuple & tuple, In
     if (! find_codec (ic.get (), & cinfo))
         return false;
 
+    ScopedContext context (cinfo);
+    if (LOG (avcodec_open2, context.ptr, cinfo.codec, nullptr) < 0)
```

Comment:
Not a fan of doing a stream-open just to get the channel count

Reply:
I don't see any other way to get information about channels here, but I'm not familiar with ffmpeg API. As alternative, changes for this file may be dropped.

Reply:
Let's do that for now, just leave this one out.